lib/src/middleware/system_clipboard.dart

```diff
@@ -127,13 +130,44 @@ void put_strand_info_on_clipboard(Store<AppState> store) {
       helices_view_order = group.helices_view_order.toList();
     }
 
+    //Get modifcations from all strands
+    var modifications = all_modifications(strands.toBuiltList());
```

Comment:
Any particular reason the `BuiltSet<Strand>` is being converted to a `BuiltList<Strand>` here? If there's a specific reason, comment it, otherwise you could just change the type of the parameter of `all_modifications` to be `Iterable<Strand>` or `BuiltSet<Strand>`.